kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -129,6 +147,49 @@ private void setup(Map<String, Object> conf) {
     this.executor = MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workerThreadCount));
   }
 
+  /**
+   * Validate an individual cluster's configuration. A cluster's configuration is valid if:<br>
```

Comment:
you don't need the `<br>` tags; they just end up making this hard to read, particularly since we don't generate any javadocs